feed/views.py

```diff
@@ -1,60 +1,184 @@
+                        'feed_main_image' : feed.imageurl_set.values('image_url').first()
+                        },
     
-        for feed in Feed.objects.all().order_by('-id'):
-            feed_data = {
-                'feed_basic_data' : {
-                    'feed_id'         : feed.id, 
-                    'feed_user'       : feed.user.nickname,
-                    'created_at'      : feed.created_at,
-                    'description'     : feed.description,
-                    'like_number'     : feed.like_number,
-                    'tag_item_number' : feed.tag_item_number,
-                    'feed_main_image' : feed.imageurl_set.values('image_url').first()
-                    },
-
-                'product_data'   : {
-                    'product_id'     : feed.product_id,
-                    'product_name'   : feed.product.name,
-                    'discount_rate'  : feed.product.discount_rate,
-                    'price'          : feed.product.price,
-                    'product_image'  : feed.product.productimageurl_set.get(is_main=1).image_url
-                    } if feed.product_id else {
-                    'product_id'     : '',
-                    'product_name'   : '',
-                    'discount_rate'  : '',
-                    'price'          : '',
-                    'product_image'  : ''
-                    },
-
-                'feed_comment_data'  : {
-                    'feed_comment_count' : feed.comment_set.count(),
-                    'comment_list'       : [{
-                            'user'      : User.objects.get(id = comment.user_id).nickname,
-                            'content'   : comment.content
-                        } for comment in feed.comment_set.all().order_by('-created_at')[:MAXIMUM_COMMENT]
-                        ] if feed.comment_set.exists() else
-                        [{
-                            'user':'',
-                            'content':''
-                        }]
+                    'product_data'   : {
+                        'product_id'     : feed.product_id,
+                        'product_name'   : feed.product.name,
+                        'discount_rate'  : feed.product.discount_rate,
+                        'price'          : feed.product.price,
+                        'product_image'  : feed.product.productimageurl_set.get(is_main=1).image_url
+                        } if feed.product_id else {
+                        'product_id'     : '',
+                        'product_name'   : '',
+                        'discount_rate'  : '',
+                        'price'          : '',
+                        'product_image'  : ''
+                        },
+    
+                    'feed_comment_data'  : {
+                        'feed_comment_count' : feed.comment_set.count(),
+                        'comment_list'       : [{
+                                'user'      : User.objects.get(id = comment.user_id).nickname,
```

Comment:
```suggestion
                                'user'      : comment.user.nickname,
```
```suggestion
                                'user'      : comment.user.nickname,
```
여기 채린님 부분이었던 것 같긴 한데, 지금 보여서 같이 수정해주시면 좋을 것 같아요!

Reply:
넵 감사합니다.